src/preload.py

```diff
@@ -955,8 +956,20 @@ def get_partition(name, image=None):
 if __name__ == "__main__":
     update_ca_certificates(**SH_OPTS)
     for line in sys.stdin:
-        data = json.loads(line)
-        method = methods[data["command"]]
-        result = method(**data.get("parameters", {}))
-        print(json.dumps({"result": result}))
+        try:
+            data = json.loads(line)
+            method = methods[data["command"]]
+            result = method(**data.get("parameters", {}))
+            response = {
+                "result": result,
+                "statusCode": 0,
+            }
+        except BaseException:
```

Comment:
Alexis suggested that `Exception` should be sufficient, but I'm still minded on `BaseException` because of this hierarchy list:
https://docs.python.org/3.6/library/exceptions.html#exception-hierarchy

Basically I wanted to ensure that absolutely any exception will be reported to the CLI process, causing it to terminate, to avoid the bad UX of infinite spinning wheels.